### DISCUSSION TITLE
Significance of "BAohbmd6aG91IFR1"?

### DISCUSSION THREAD

**Discussion by Frielyy** [ct-Open-Source/tuya-convert#961] [Q&A]
Hey Colin/guys,

I know I might be a bit unlikely to get a reply here, but that's ok.

I really would like to know the significance of the string "BAohbmd6aG91IFR1", which returns little but this page in Google searches but I see it regularly in Tuya Smart packets. It's for my dissertation anyhow.

I notice in particular you have it on this page -- https://github.com/ct-Open-Source/tuya-convert/blob/master/scripts/psk-frontend.py

Cheers!

**Comment by kueblc** [comment]
As far as I can tell, it's a fixed, pseudo random string they use somewhat like a salt in their encryption scheme. It prefixes every PSK ID sent by the device to the cloud during the PSK handshake. It has no significance other than that it's what they've chosen.

PS. Good luck on your dissertation, would be interested to see!
